schemas.py

```diff
@@ -43,18 +50,20 @@ class Config:
         orm_mode = True
 
 # base schema for user data
+
+
 class UserInfoBase(BaseModel):
```

Comment:
I see you add "user_level" so you have  to add it also in the class UserInfoBase